packages/zowe-explorer/__tests__/__unit__/shared/actions.unit.test.ts

```diff
@@ -110,9 +111,6 @@ async function createGlobalMocks() {
     return globalMocks;
 }
 
-// Idea is borrowed from: https://github.com/kulshekhar/ts-jest/blob/master/src/util/testing.ts
-const mocked = <T extends (...args: any[]) => any>(fn: T): jest.Mock<ReturnType<T>> => fn as any;
```

Comment:
no problem! 😋 
thanks for reviewing - for the sake of stability, I'm going to mark it as draft. I noticed that some of the comparisons may not always be true in imperative because of how we store the schema in the cache.

I will update this PR as soon as this is resolved 😅👍